### PR TITLE
P17 cruise info update

### PR DESCRIPTION
**Pull request by rtravis-whoi**
updated P17 cruise start/end dates/times per Sheri's slack msg:
"End of cruise 2021-11-15 12:45:00 UTC.
I think Start was 2021-10-29 14:45:00 UTC." (RGT confirmed via elog entry)

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -264,4 +264,4 @@ TS20210809,Tenacious II,2021-08-09T01:00:00,2021-08-09T23:59:59,Pioneer gliders
 TS20210903,Tenacious II,2021-09-03T14:00:00,2021-09-04T10:00:00,Pioneer 16d glider deployments
 TN394,Thomas G. Thompson,2021-09-08T21:00:00,2021-09-22T21:50:00,Endurance 15 moorings profilers and gliders
 WJ20211020,Warren Jr.,2021-10-20T15:00:00,2021-10-22T09:00:00,Pioneer glider cp_564 recovery
-AR61,Neil Armstrong,2021-10-26T00:00:00,2021-11-15T23:59:00,Pioneer 17
\ No newline at end of file
+AR61,Neil Armstrong,2021-10-29T14:45:00,2021-11-15T12:45:00,Pioneer 17
```
Comment (cdobs) [inline cruise/CruiseInformation.csv:267]: Looks good

**Review by cdobs — APPROVED** [review]

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (batryn) [reply cruise/CruiseInformation.csv:267]: Looks good

**Review by batryn — APPROVED** [review]
Looks good

File: vessel/shiplist.csv [hunk 2]
```diff
@@ -40,6 +40,7 @@ AR,R/V,Neil Armstrong,33VB,WHOI,WARL,338767000,9688946,72.5,7,4.6,AGOR-27,Y,Y
 NH,R/V,New Horizon,32NM,SIO,WKWB,303273000,7723821,51.8,6.3,3.5,,(N),Y
 OC,R/V,Oceanus,32OC,OSU,WXAQ,367977000,7603617,53.9,6.2,6,,Y,Y
 EX,R/V,Okeanos Explorer,334A,OMAO,WTDH,369888000,8835114,68.3,6.2,5.2,,Y,Y
+SU,R/V,Pacific Surveyor,,,,,,,,,,,
```
Comment (cdobs) [inline vessel/shiplist.csv:43]: looks good

File: cruise/CruiseInformation.csv [hunk 3]
```diff
@@ -264,4 +264,5 @@ TS20210809,Tenacious II,2021-08-09T01:00:00,2021-08-09T23:59:59,Pioneer gliders
 TS20210903,Tenacious II,2021-09-03T14:00:00,2021-09-04T10:00:00,Pioneer 16d glider deployments
 TN394,Thomas G. Thompson,2021-09-08T21:00:00,2021-09-22T21:50:00,Endurance 15 moorings profilers and gliders
 WJ20211020,Warren Jr.,2021-10-20T15:00:00,2021-10-22T09:00:00,Pioneer glider cp_564 recovery
-AR61,Neil Armstrong,2021-10-26T00:00:00,2021-11-15T23:59:00,Pioneer 17
\ No newline at end of file
+AR61,Neil Armstrong,2021-10-29T14:45:00,2021-11-15T12:45:00,Pioneer 17
+SU20211112,Pacific Surveyor,2021-11-12T23:00:00,2021-11-13T16:00:00,Papa 8a Glider Deployments
```
Comment (cdobs) [inline cruise/CruiseInformation.csv:268]: looks good

**Review by cdobs — APPROVED** [review]

**Review by batryn — APPROVED** [review]
Looks good

**Comment by rtravis-whoi** [comment]
[Pioneer 17 AM Updates [DT-151]](https://trello.com/c/LYsz2Yeu/151-pioneer-17-am-updates-dt-151)

**Pull request merged**